channels/us.m3u

```diff
@@ -3564,3 +3564,21 @@ http://wttv-lh.akamaihd.net:80/i/WTTVBreaking_1@333494/master.m3u8
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="Mohabat TV" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/tucTC1O.jpg" group-title="",Mohabat TV
 http://media.mohabat.tv:1935/live_transcoder/ngrp:mohabat.stream_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Ind TV USA
+https://t06858-lh.akamaihd.net/i/t06858a_1@719164/index_1500000_av-p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Jasmin channel 18+ HD
+http://109.71.162.112:1935/live/hd.jasminchannel.stream/chunklist_w1793148297.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Jasmin TV
+http://109.71.162.112:1935/live/hd.jasminchannel.stream/chunklist_w359453841.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="XXX",Eroxxx HD
+http://147.78.163.247:8000/play/a021
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/KOgoXoG.png" group-title="",MCN6
+https://d18fcxaqfnwjhj.cloudfront.net/CDN_Ingest/MCN6_MAIN.smil/Playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/KOgoXoG.png" group-title="",MCN6 Arts Channel
+https://d18fcxaqfnwjhj.cloudfront.net/CDN_Ingest/MCN6_COMEDY.smil/Playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/KOgoXoG.png" group-title="",MCN6 Music Channel
+https://d18fcxaqfnwjhj.cloudfront.net/CDN_Ingest/MCN6_MUSIC.smil/Playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",ASS TV
+http://www.ast.tv/stream/2/master.m3u8?fluxuslust.m3u8
```

Comment:
At the moment, links to AST TV channel are placed in the Cypriot playlist: https://github.com/iptv-org/iptv/pull/2146

Reply:
AST's legal data says the society behind it is Chypriot.

Reply:
![image](https://user-images.githubusercontent.com/30985701/117485379-966b3500-af68-11eb-9cfe-cf986d969b24.png)

However, the host is indeed American, and the IP comes from... the Netherlands. A complete mix of informations there.

Reply:
aaaand.... the website's terms of services says they're Chypriot... 
![image](https://user-images.githubusercontent.com/30985701/117485715-05e12480-af69-11eb-8676-f00b7fddf568.png)
![image](https://user-images.githubusercontent.com/30985701/117485733-0b3e6f00-af69-11eb-8acc-0a49dca300bc.png)